### ISSUE TITLE
Autocomplete stoped working after upgrade to v1.5.2

### ISSUE THREAD

**Issue by fatonh** [ajaxorg/ace#4820]
### Describe the bug

After upgrading to version 1.5.2 autocomplete has started working. Im using 

### Expected Behavior

Autocomplete will pop up soon after starting writing CSS code

### Current Behavior

Not displaying autocomplete 

### Reproduction Steps

Snipped code from option enabled for ACE editor
 
```
editor.setOptions({
                enableBasicAutocompletion: true,
                enableSnippets: true,
                enableLiveAutocompletion: true,
                autoScrollEditorIntoView: false
            });
```

### Additional Information/Context

Seems that the changes that have been made here https://github.com/ajaxorg/ace/compare/v1.5.1...v1.5.2#diff-e902cbdb49877208b53c4c8d7a36397df12ddf0e92f3f415366de3c8106b221aL354 was the cause for this. Something with z-index maybe

### Ace Version

v.1.5.2

**Comment by andrewnester** [comment]
@fatonh do you have some custom styles which you use for the editor?

**Comment by fatonh** [comment]
@andrewnester No, I don't use any custom styles for editor. Theme that I use tho is chrome, if that helps

**Comment by andrewnester** [comment]
@fatonh Can you try to upgrade to latest 1.5.3 version?

**Comment by andrewnester** [comment]
As I'm not able to reproduce it locally and in kitchen sink (which you also confirmed), there might be something specific to your setup, so we need to dig into that. Could you please check if you get any errors in browser console for version 1.5.2 and 1.5.3?
Also, can you share a full file with Ace setup if possible, so we can reproduce it.

Thanks!

**Comment by BenSouchet** [comment]
@fatonh Hi, is you project accessible on the web ? If yes, can you share the url ? So we can check.
Ideas I have in mind: maybe the container of the Ace block you set is in `position: absolute` or have already a big `z-index`.

**Comment by fatonh** [comment]
@andrewnester There is no error in browser console and here I share part of the setup for editor, hopefully will help

`var editors = $('#col-content').find('.ace-editor');

        editors.each(function (i, e) {
            var editorElement = e;
            var x = 0;
            var y = 0;

            $(editorElement).off('click').click(function (e) {
                x = e.pageX;
                y = e.pageY;
            });

            ace.require("ace/ext/language_tools");
            ace.edit('scss-editor').resize(true);

            var editor = ace.edit(editorElement);
            editor.setShowPrintMargin(false);
            editor.setHighlightActiveLine(true);
            editor.getSession().setUseWrapMode(true);
            editor.renderer.setHScrollBarAlwaysVisible = true;
            editor.setTheme("ace/theme/chrome");
            editor.getSession().setMode($(this).attr('data-ace-lang') ? $(this).attr('data-ace-lang') : 'ace/mode/css');
            editor.setOptions({
                enableBasicAutocompletion: true,
                enableSnippets: true,
                enableLiveAutocompletion: true,
                autoScrollEditorIntoView: false
            });

            if (window.localStorage) {
                // Sets up button-click listeners
                var addEventQueueButtonListeners = function () {
                    $('.btn-undo').on('click', function (event) {
                        event.preventDefault();
                        var um = editor.getSession().getUndoManager();
                        if (um.hasUndo()) {
                            um.undo();
                        }
                    });

                    $('.btn-redo').on('click', function (event) {
                        event.preventDefault();
                        var um = editor.getSession().getUndoManager();
                        if (um.hasRedo()) {
                            um.redo();
                        }
                    });

                    editor.on('input', function (e) {
                        var um = editor.getSession().getUndoManager();
                        $('.btn-undo').attr('disabled', !um.hasUndo());
                        $('.btn-redo').attr('disabled', !um.hasRedo());
                    });
                };

                if ($(e).attr('id') === 'scss-editor') {
                    addEventQueueButtonListeners();
                }
            } else {
                console.log('No localStorage - editor redo/undo not overridden');
            }

            // Center the the cursor where it was before creating, only apply to first editor
            if (getSessionStorage(projectid) !== -1 && $('#scss-editor').length > 0 && editor.id === 'editor1') {
                var cursorPosition = JSON.parse(getSessionStorage(projectid));
                if (typeof cursorPosition !== "undefined" && cursorPosition !== 1) {
                    editor.selection.setRange(cursorPosition, true);
                    // Have to wait before it actually moves
                    setTimeout(function () {
                        editor.resize(true);
                        editor.scrollToLine(cursorPosition.end.row, true, true, function () {});
                        editor.focus();
                    }, 400);
                }
            }
        });`

**Comment by fatonh** [comment]
@BenSouchet Hi, there is a need for login so a bit trouble to share that :/ 
There is no `position: absolute;` for the container where we set ace editor, tho there is some `height: 80vh;
    overflow-y: scroll;`. Moreover, when I remove the `position: relative;` from ace_editor then the autocomplete tool shows up but mess up the styling of the application. So yes something with styling is happening here. 

Before and after removing the `position: relative;` from .ace_editor class
![Screenshot 2022-06-08 at 11 27 39](https://user-images.githubusercontent.com/3781157/172582870-8f0cb3b0-bea2-4637-a991-e7a8c432afe3.png)
![Screenshot 2022-06-08 at 11 27 47](https://user-images.githubusercontent.com/3781157/172582896-a21a5361-4694-4ffc-b2e1-218c59b5c4d2.png)

**Comment by andrewnester** [comment]
As @nightwing mentioned in private conversation 
> the selector from .ace_editor.ace_autocomplete { to .ace_autocomplete 
so position fixed for the popup got lower specificity

I published a PR to actually restore the styling for code completion popup

**Comment by fatonh** [comment]
@andrewnester Thank you for ridiculous fast actions :)

**Comment by andrewnester** [comment]
Released in 1.6.0
